vespa/test_integration_docker.py

```diff
@@ -141,3 +150,125 @@ def tearDown(self) -> None:
         shutil.rmtree(self.disk_folder, ignore_errors=True)
         self.vespa_docker.container.stop()
         self.vespa_docker.container.remove()
+
+
+class TestOnnxModelDockerDeployment(unittest.TestCase):
+    def setUp(self) -> None:
+        #
+        # Create application package
+        #
+        self.app_package = ApplicationPackage(name="cord19")
+        self.app_package.schema.add_fields(
+            Field(name="cord_uid", type="string", indexing=["attribute", "summary"]),
+            Field(
+                name="title",
+                type="string",
+                indexing=["index", "summary"],
+                index="enable-bm25",
+            ),
+            Field(
+                name="doc_token_ids",
+                type="tensor<float>(d0[96])",
+                indexing=["attribute", "summary"],
+            ),
+        )
+        self.app_package.schema.add_field_set(
+            FieldSet(name="default", fields=["title"])
+        )
+        self.app_package.query_profile_type.add_fields(
+            QueryTypeField(
+                name="ranking.features.query(query_token_ids)",
+                type="tensor<float>(d0[32])",
+            )
+        )
+        self.app_package.schema.add_model(
+            OnnxModel(
+                model_name="bert_tiny",
+                model_file_path=os.path.join(
+                    os.getenv("RESOURCES_DIR"), "bert_tiny.onnx"
+                ),
+                inputs={
+                    "input_ids": "input_ids",
+                    "token_type_ids": "token_type_ids",
+                    "attention_mask": "attention_mask",
+                },
+                outputs={"logits": "logits"},
+            )
+        )
+        self.app_package.schema.add_rank_profile(
+            RankProfile(
+                name="bert",
+                inherits="default",
+                constants={"TOKEN_NONE": 0, "TOKEN_CLS": 101, "TOKEN_SEP": 102},
+                functions=[
+                    Function(
+                        name="question_length",
+                        expression="sum(map(query(query_token_ids), f(a)(a > 0)))",
+                    ),
+                    Function(
+                        name="doc_length",
+                        expression="sum(map(attribute(doc_token_ids), f(a)(a > 0)))",
+                    ),
+                    Function(
```

Comment:
Vespa now has a `tokenInputIds` function that does this. Can be replaced with that.

Reply:
Documentation for that is on https://docs.vespa.ai/en/reference/rank-features.html

Reply:
I will change the expressions to `tokenInputIds` in a new PR once I get confirmed the [bug fix](https://github.com/vespa-engine/vespa/pull/16253) got released and worked as expected.